src/editor/EditorControls.tsx

```diff
@@ -3,22 +3,40 @@ import {
   ToggleMarkButton,
   LinkButton,
   InlineLatexButton,
+  ToggleBlockButton,
+  ExtrasDropdown,
+  EditorDropdown,
+  ToggleMarkButtonText,
 } from "./components/controls";
-import { Mark } from "./types";
+import { Mark, BlockElementType } from "./types";
 
 import "./styles/EditorControls.scss";
 
 const EditorControls: React.FC = () => {
   return (
     <div className="editor-controls">
-      <ToggleMarkButton icon="bold" mark={Mark.Bold} />
-      <ToggleMarkButton icon="italic" mark={Mark.Italic} />
-      <ToggleMarkButton icon="underline" mark={Mark.Underline} />
+      <EditorDropdown
+        rootButton={<ToggleMarkButton icon="bold" mark={Mark.Stress1} />}
+        id="stress-dropdown"
+      >
+        <ToggleMarkButtonText text="Stress 2" mark={Mark.Stress2} />
+      </EditorDropdown>
+      <EditorDropdown
+        rootButton={<ToggleMarkButton icon="italic" mark={Mark.Emph1} />}
+        id="emph-dropdown"
+      >
+        <ToggleMarkButtonText text="Emphasis 2" mark={Mark.Emph2} />
+        <ToggleMarkButtonText text="Emphasis 3" mark={Mark.Emph3} />
+        <ToggleMarkButtonText text="Emphasis 4" mark={Mark.Emph4} />
+      </EditorDropdown>
       <ToggleMarkButton icon="strikethrough" mark={Mark.Strikethrough} />
       <ToggleMarkButton icon="code" mark={Mark.Code} />
       <div className="editor-separator" />
       <LinkButton />
       <InlineLatexButton />
+      <div className="editor-separator" />
+      <ToggleBlockButton icon="heading" blockType={BlockElementType.Header} />
```

Comment:
This should be a dropdown for both heading styles.

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/68/commits/b664fba667ae40e03a7009134ff8b8ce381f3ffa